modules/snooker/src/main/ts/ephox/snooker/api/TableLookup.ts

```diff
@@ -44,6 +46,8 @@ const row = (element: SugarElement, isRoot?: (e: SugarElement) => boolean) => lo
 
 const rows = (ancestor: SugarElement): SugarElement<HTMLTableRowElement>[] => LayerSelector.firstLayer(ancestor, 'tr');
 
+const columnGroups = (ancestor: SugarElement): SugarElement<HTMLElement>[] => LayerSelector.firstLayer(ancestor, 'colGroup');
```

Comment:
`colGroup` is not the element name, this should be `colgroup`.

Reply:
Dammit, I thought I'd changed all of those.